### DISCUSSION TITLE
Highlight of keywords is not working in Markdown

### DISCUSSION THREAD

**Discussion by tobiaslombacher** [folke/todo-comments.nvim#216] [General]
I have set up the plugin with the standard configuration in Neovim 0.9. Neovim checkhealth does not show errors. 

Within lua files the plugin works as expected and it highlights the keywords in comments
e.g. -- TODO:  blablablalbalbla

However within markdown files it does not highlight the keywords. I have not changed any of the parameters within the  opt{ .... }

I have tried 

- the keyword alone: TODO:
- the keyword without colon: TODO
- with the markdown comment convention: [comment:] TODO:
- within the brackets [TODO:]

but it does not work in markdown files. 

Is there anything I am missing? Do I need to enable it for markdown files specifically? Am I using the wrong syntax? 

What I would like to do in my markdown files is to use the keyword:

 TODO: action item 

within my markdown files and have them highlighted. 

Help would be highly appreciated.

**Comment by folke** [comment]
check the treesitter setting. todo comments only work in comment sections of files

**Reply by tobiaslombacher** [reply]
Ok, after many trials and a clean setup I have found a incorrect bracket in my init.lua which caused the plugin not to work properly. Now it is working as expected. Thank you very much for your suggestion.
